src/CommunityToolkit.Maui/Controls/Avatar.cs

```diff
@@ -0,0 +1,557 @@
+using System.ComponentModel;
+using Microsoft.Maui.Controls.Internals;
+using Microsoft.Maui.Controls.Shapes;
+
+namespace CommunityToolkit.Maui.Controls;
+
+/// <summary>Avatar content view.</summary>
+public class Avatar : ContentView, IAvatarElement, IFontElement, ITextElement, IImageElement
```

Comment:
Please name this `AvatarView`

```suggestion
public class AvatarView : ContentView, IAvatarElement, IFontElement, ITextElement, IImageElement
```

Reply:
I'm not personally 100% that it is necessary or right to add "view" to the name of the control.  This is more like the "Button" or "Label" control but understand that it is also a member of the "Present data" subclass.  I will make the change if it is agreed to call this "AvatarView".

Reply:
I have renamed to AvatarView. I believe I have resolved this issue.